hawkular-alerts-rest-tests/src/test/groovy/org/hawkular/alerts/rest/AbstractITestBase.groovy

```diff
@@ -30,14 +34,18 @@ class AbstractITestBase {
 
     static baseURI = System.getProperty('hawkular.base-uri') ?: 'http://127.0.0.1:8080/hawkular/alerts/'
     static RESTClient client
+    static String tenantHeaderName = "Hawkular-Tenant"
     static testTenant = "28026b36-8fe4-4332-84c8-524e173a68bf"
+    static final String TENANT_PREFIX = UUID.randomUUID().toString()
+    static final AtomicInteger TENANT_ID_COUNTER = new AtomicInteger(0)
 
     @BeforeClass
     static void initClient() {
 
         client = new RESTClient(baseURI, ContentType.JSON)
         // this prevents 404 from being wrapped in an Exception, just return the response, better for testing
         client.handler.failure = { it }
+        /*
```

Comment:
Yes, ITests should be fine.  I left the commented code in case we wanted to use it in the future for any reason.